src/fondant/pipeline/runner.py

```diff
@@ -126,3 +126,32 @@ def get_name_from_spec(self, input_spec: str):
         with open(input_spec) as f:
             spec = yaml.safe_load(f)
             return spec["pipelineInfo"]["name"]
+
+
+class SagemakerRunner(Runner):
+    def __init__(self):
+        self.__resolve_imports()
+
+    def __resolve_imports(self):
+        import boto3
+
+        self.boto3 = boto3
+
+    def run(self, input_spec: str, *args, **kwargs):
+        """Run a sagemaker pipeline."""
+        client = self.boto3.client("sagemaker")
```

Comment:
We need to use the lower level SDK to start a pipeline from a spec. There is no support for this in the python SDK